src/content/developers/docs/nodes-and-clients/light-clients/index.md

```diff
@@ -0,0 +1,59 @@
+
+On the execution layer there is no single specification for a light execution client. The scope of a light execution client can vary from a "light mode" of a full execution client that has all the EVM and networking functionality of a full node but only verifies block headers, without downloading the associated data, or it can a more stripped down client that relies heavily upon forwarding requests to an RPC provider to interact with Ethereum.
+
+## Why are light clients important? {#why-are-light-clients-important}
+
+Light clients matter because they allow users to verify incoming data rather than blindly trusting that their data provider is correct and honest. The data they receive can be checked against block headers that they know have been signed by at least 2/3 of a random set of 512 Ethereum validators. This is very strong evidence that the data is correct, and it doesn't require trusting some specific provider to be honest. The light client requires a tiny amount of computing power, memory and storage so it can be run on a mobile phone, embedded in an app or as part of a browser. Light clients therefore offer trust-minimized access to Ethereum without the need for expensive hardware. Light clients are a way to make trustless access to Ethereum just as frictionless as trusting a third-party provider.
+
+Let's take a simple example. Imagine you want to check your account balance. To do this you have to make a request to an Ethereum node. That node will check it's local copy of the Ethereum state for your balance and return it to you. If you don't have direct access to a node, there are centralized operators that provide this data as a service. You can send a request to them, they check their node, and send the result back to you. The problem with this is that you then have to trust the provider to be giving you the correct information. You can never really know the information is correct if you can't verify it for yourself.
+
+A light client addresses this issue. You still request data from some external provider, but when you receive the data back it comes with a proof that your light node can check against the information it received in the block header. This means Ethereum is verifying the correctness of your data instead of some trusted operator.
+
+## What innovations do light clients enable? {#what-innovations-do-light-clients-enable}
+
+The primary benefit of light clients is enabling more people to access Ethereum independently with negligible hardware requirements and minimal reliance on third parties. This is good for users because they can verify their own data anbd it is good for the network because it increases the number and diversity of nodes that are verifying the chain.
```

Comment:
Should this sentence be in this section? Feels a bit repetitive from the last section, and seem to fit more in "What are they important" than "What innovations do they enable".

If we decide to keep it, small typo:
```suggestion
The primary benefit of light clients is enabling more people to access Ethereum independently with negligible hardware requirements and minimal reliance on third parties. This is good for users because they can verify their own data and it is good for the network because it increases the number and diversity of nodes that are verifying the chain.
```